worker/src/main/java/gov/cms/ab2d/worker/service/FileService.java

```diff
@@ -0,0 +1,49 @@
+package gov.cms.ab2d.worker.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.nio.file.StandardOpenOption.APPEND;
+
+@Slf4j
+@Component
+public class FileService {
+
+
+    Path createDirectory(Path outputDir) {
+        Path outputDirectory = null;
+        try {
+            outputDirectory = Files.createDirectories(outputDir);
+        } catch (IOException e) {
+            final String errMsg = "Could not create output directory : ";
+            log.error("{} : {}", errMsg, outputDir.toAbsolutePath());
+            throw new RuntimeException(errMsg + outputDir.getFileName(), e);
```

Comment:
`java.io.UncheckedIOException` could be a better fit.

Reply:
done